terraform/rancher/cluster/provider.tf

```diff
@@ -35,3 +35,9 @@ provider "kubernetes" {
   cluster_ca_certificate = base64decode(data.aws_eks_cluster.cluster.certificate_authority.0.data)
   token                  = data.aws_eks_cluster_auth.cluster.token
 }
+
+provider "elasticstack" {
+  elasticsearch {
+    endpoints = [var.elasticsearch_url]
```

Comment:
This url should be generated like "${module.eks_cluster.cluster_id}-elasticsearch.${var.root_domain}"

Reply:
I've changed the URL.